staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -803,6 +803,23 @@ type KubeletConfiguration struct {
 	// Default: true
 	// +optional
 	EnableSystemLogHandler *bool `json:"enableSystemLogHandler,omitempty"`
+	// lockFilePath is the path that kubelet will use to as a lock file.
+	// It uses this file as a lock to synchronize with other kubelet processes
+	// that may be running.
+	// Dynamic Kubelet Config (beta): If dynamically updating this field, consider that
```

Comment:
we have discussed deprecating dynamic kubelet config, while we have not made progress on that in 1.20, i still think i would like to push to do that in 1.21.  is this being used in another context that i may forget?

Reply:
These fields aren't really useful with dynamic config. This comment is just a warning like we have on every field regarding dynamic config.